app/bundles/CoreBundle/Templating/Twig/Extension/MenuExtension.php

```diff
@@ -37,4 +41,47 @@ public function menuRender($menu, array $options = [], ?string $renderer = null)
     {
         return $this->menuHelper->render($menu, $options, $renderer);
     }
+
+    /**
+     * Parses attributes for the menu view.
+     *
+     * @param array<string,string> $attributes
+     * @param array<string,string> $overrides
+     */
+    public function parseMenuAttributes(array $attributes, array $overrides = []): string
+    {
+        return $this->menuHelper->parseAttributes($attributes, $overrides);
+    }
+
+    /**
+     * Concats the appropriate classes for menu links.
+     *
+     * @param array<string,string> $options
+     *
+     * @return array<mixed>
+     */
+    public function buildMenuClasses(ItemInterface $item, MatcherInterface|null $matcher, array $options, ?string $extraClasses)
```

Comment:
I'll have to get used to union types now that we support only PHP 8.0. But do you think you could use
```suggestion
    public function buildMenuClasses(ItemInterface $item, ?MatcherInterface $matcher, array $options, ?string $extraClasses)
```
as there is also `?string` in this row just to use the same style of writing the same thing in 1 row?